palisade-service/README.md

```diff
@@ -47,10 +53,9 @@ The application exposes two REST endpoints:
 
 ## Kafka Interface
```

Comment:
I think understanding the RunnableGraph is core to understanding the Kafka streaming used in this application.  I think the reader could benefit from a high level summary of the the class and especially runner method.

Reply:
The readme is only really an overview of the service, for specifics, the readme in the runnable graph should provide the relevant information